pkgs/by-name/fi/fido2luks/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["command-line-utilities"]
 license = "MPL-2.0"
 
 [dependencies]
-ctap_hmac = { version="0.4.5", features = ["request_multiple"] }
+ctap_hmac = { git="https://github.com/robin-nitrokey/ctap", rev="29f684c88b6247b3fabaaa8bb75da45d48a92c69", features = ["request_multiple"] }
```

Comment:
This change can be done via a patch, so we don't need to vendor the entire file. Is there an upstream issue/commit we can reference?

Reply:
It can be done via patch yes, but not with cargo. Unless you know a secret undocumented nixpkgs trick to patch cargo dependencies.

Reply:
ah you mean patching the Cargo.toml

Reply:
I was reviewing commit by commit and the changes make sense in the big picture only.

Reply:
done
i also removed the nitrokey fix, since it didn't work anyway